components/composed/model/ModelGrid/ModelGridItem.tsx

```diff
@@ -30,12 +30,13 @@ function ModelGridItem<T extends Record<string, unknown>>({ row }: Props<T>) {
 
           return (
             <div key={i}>
-              {cell.column.id !== "title" && cell.column.id !== "name" && (
-                <span>
-                  {cell.render("Header")}
-                  {`: `}
-                </span>
-              )}
+              {cell.column.id.toLowerCase() !== "title" &&
+                cell.column.id !== "name" && (
```

Comment:
Need to check against `.toLowerCase` for "name" as well, and then you're all set!